intro-HPC/leuven/available-modules.tex

```diff
@@ -0,0 +1,102 @@
+\begin{prompt}
```

Comment:
What's the intention of including this?

Keeping this up-to-date is a nightmare...

Our module tree is updated *daily* in Ghent...